sphinxcontrib/needs/utils.py

```diff
@@ -90,18 +93,7 @@ def row_col_maker(app, fromdocname, all_needs, need_info, need_key, make_ref=Fal
     return row_col
 
 
-def status_sorter(a):
```

Comment:
See comments above, not sure if we still need it

Reply:
don't think we need it, i've changed the above to

```python
all_needs = sorted(all_needs, key=lambda node: node[sort_key] or "")
```

does that do what you're after?

Reply:
Yes, thanks. Looks good.